index.html

```diff
@@ -134,12 +134,16 @@ <h2></h2>
           <p class="keywords"></p>
```

Comment:
first off: really nice work and thanks for updating the preview. I just clicked and saw it instantly. It works well. Cudos!

Yet, there is a design glitch (maybe only safari). The articles are much more narrow now and the placeholder below is wider then the articles. can you fix this?